panels/dev-info/ha-panel-dev-info.html

```diff
@@ -54,6 +58,56 @@
         clear: both;
         white-space: pre-wrap;
       }
+
+      .system-log-intro {
+        margin-top: 16px;
+        border-top: 1px solid var(--light-primary-color);
+        padding-top: 16px;
+      }
+
+      paper-card {
+        display: block;
+        padding-top: 16px;
+      }
+
+      paper-item {
+        cursor: pointer;
+      }
+
+      .header {
+        @apply(--paper-font-title);
+      }
+
+      .row {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      paper-dialog {
+        border-radius: 2px;
+      }
+
+      @media all and (max-width: 450px) {
```

Comment:
When the max-width is 450px, change the padding off the content on the sides to 0 so the cards take the full width.

Reply:
Same here.